### ISSUE TITLE
wrong routing

### ISSUE THREAD

**Issue by Kallirroi** [Kallirroi/wordsinspace#115]
![image](https://user-images.githubusercontent.com/10440556/91748843-9ec7d080-eb8e-11ea-90bf-320ef001b10e.png)

**Comment by samtous** [comment]
## all the redirect issues

In general: netlify redirect rules _worked_ as they work like server redirect rules. Pretty sure there might be a way to get server redirects working again (maybe using the plugin?).

### pages
Everything on shannon's site is currently at /shannon/:date/:slug, and old links will 404 if we don't have a redirect rule 

Things that used to be at /presentations/:slug are now at /:slug, so we need a redirect to go from there

### static files
These are tricky as they still need to live on the server, so when someone requests a single file (not a page) we need to redirect to the new install, icd.wordsinspace.net. Examples of this are:

-- her folder of password-protected pdfs (/secure/)
-- all image files

she actively uses these links quite a lot. These aren't internal redirects in Gatsby, and I think need to return a 301 to work properly.

**Comment by samtous** [comment]
Resolved. Occasionally I see things like:

![Screen Shot 2020-09-02 at 13 36 33](https://user-images.githubusercontent.com/5625731/92019806-2f381980-ed25-11ea-8be1-60bd8d91298a.png)

Which I believe is gatsby pre-emptively getting files at the other end of links on the page. The reason this was happening before wasn't because the in-page category icons were incorrectly linking to `/work/publications`, instead of `/publications/`... which was fixed in `categoryLink.js`. Not sure where these double slugs are coming from now; there's only ever about 1 per page so pretty sure it's just 1 link.

anyway tl;dr this _is_ a routing issue, but it's not caused by redirects, which are back in and working fine
